### PR TITLE
[musiclibrary] add SortByArtistThenYear

### PR DESCRIPTION
**Pull request by mkortstiege**
This removes `albumssortbyartistthenyear` from advancedsettings.xml and adds it as regular sort option. Requested by @MartijnKaijser

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
tested and worked \o/

**Comment by MartijnKaijser** [comment]
jenkins build this please

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -2543,7 +2543,12 @@ msgctxt "#577"
 msgid "Date taken"
 msgstr ""
 
-#empty strings from id 578 to 579
+#: xbmc/utils/SortUtils.cpp
+msgctxt "#578"
+msgid "Artist / Year"
```
Comment (uNiversaI) [inline addons/resource.language.en_gb/resources/strings.po]: Sort: Artist / Year

Comment (MartijnKaijser) [inline addons/resource.language.en_gb/resources/strings.po]: No

**Comment by mkortstiege** [comment]
Any objections to shove this one in?

**Comment by MartijnKaijser** [comment]
@NedScott please adjust wiki page

**Pull request merged**